rocketchat-core/src/main/java/com/rocketchat/core/callback/RoomCallback.java

```diff
@@ -18,4 +19,7 @@
         void onGetRoomMembers(Integer total, List<User> members);
     }
 
-}
+    public interface GetFilesCallback extends Callback {
```

Comment:
take a look on `PaginatedCallback<T>` it is not being used yet, so you can tweak it for this...

Reply:
Nice!